internal/compiler-bridge-test/src/test/scala/sbt/internal/inc/ExtractUsedNamesPerformanceSpecification.scala

```diff
@@ -23,8 +26,22 @@ class ExtractUsedNamesPerformanceSpecification extends CompilingSpecification {
   }
 
   val TestResource = "/ExtractUsedNamesPerformance.scala.source"
-  // Some difference between 2.10, 2.11, and 2.12
-  val scalaDiff = Set("Any", "Nothing", "_root_", "StringAdd", "Option")
+  // Some difference between 2.10, 2.11, 2.12, and 2.13
+  val scalaDiff = Set(
+    "Any",
+    "Nothing",
+    "_root_",
+    "StringAdd",
+    "Option",
+    "?0",
+    "package",
+    "acme",
+    "ModuleSerializationProxy",
+    "scala;runtime;ModuleSerializationProxy;init;",
+    "Class",
+    "Sealed"
```

Comment:
This is most likely the reason why we don't track incrementality for sealed class hierarchies anymore in Scala 2.13. Not as severe correctness-wise as the other error, but still worth fixing...

Reply:
The capitalized Class and Sealed are new named that started to show up in 2.13, but didn't exist before.
We probably should filter it out so it doesn't lead to overcompilation.